src/builtin-addons/core/template-completion-provider.ts

```diff
@@ -344,7 +345,7 @@ export default class TemplateCompletionProvider {
     }
 
     if (this.hasNamespaceSupport) {
-      const registry = this.server.getRegistry(this.project.root);
+      const registry = this.server.getTopLevelRootRegistry(this.project.root);
```

Comment:
in my understanding, it's not nessesary to have "blind" parent lookup.
all we have to do - is collect all files, from known roots (for project)
and it will be like: 

```js
const roots = [project.root, ...project.addonsMeta.map((meta)=>meta.root))]
```

and we could improve `getRegistry` function to support array (as argument), and we will return all items, based on all knwon roots.

Reply:
suport for multiple roots added in https://github.com/lifeart/ember-language-server/pull/215

Reply:
multiple roots now could be found on `project.roots` https://github.com/lifeart/ember-language-server/pull/216

Reply:
After trying the multiple roots, I am seeing a performance issue when we are using 
```const roots = [project.root, ...project.addonsMeta.map((meta)=>meta.root))]
``` (~1.2 seconds)
Also, we are filtering inside of a filter O(m*n) :(
We should think about making this performant, else larger apps will get a very slow feedback